french/script/story/st_00.rpy

```diff
@@ -6,901 +6,901 @@ translate french s00_last_time_on_jpde_265b952a:
 
 # game/script/story/st_00.rpy:507
 translate french battle_00_02_end_f7e45933:
 
     # adt sun amused "Goodbye, Sienna."
-    adt sun amused ""
+    adt sun amused "Adieu, Sienna."
 
 # game/script/story/st_00.rpy:508
 translate french battle_00_02_end_69a580cf:
 
     # d_i sun angry sweat "(Oh no, you don't!)"
-    d_i sun angry sweat ""
+    d_i sun angry sweat "(Oh non, certainement pas !)"
 
 # game/script/story/st_00.rpy:509
 translate french battle_00_02_end_61083fd7:
 
     # d sun shock sweat "{b}ROCKET PUNCH!"
-    d sun shock sweat ""
+    d sun shock sweat "{b}ROCKET PUNCH!"
 
 # game/script/story/st_00.rpy:511
 translate french battle_00_02_end_29a29ce7:
 
     # adt surprise "Rocket wha-"
-    adt surprise ""
+    adt surprise "Poing fus-"
 
 # game/script/story/st_00.rpy:526
 translate french battle_00_02_end_fac2746b:
 
     # adt nulla "You...{w=0.5} Vermin...!!!"
-    adt nulla ""
+    adt nulla "Toi...{w=0.5} Vermine... !!!"
 
 # game/script/story/st_00.rpy:527
 translate french battle_00_02_end_dad0e5a5:
 
     # d_i nulla "(Hah!{w=0.3} Right in the chin!)"
-    d_i nulla ""
+    d_i nulla "(Ah !{w=0.3} Droit dans le menton !)"
 
 # game/script/story/st_00.rpy:529
 translate french battle_00_02_end_00e2efb8:
 
     # d_i nulla "(Wait...{w=0.5} His eyes...?!{w=0.3} You gotta be kidding me!)"
-    d_i nulla ""
+    d_i nulla "(Attends...{w=0.5} Ses yeux... ?!{w=0.3} Vous vous foutez de moi !)"
 
 # game/script/story/st_00.rpy:530
 translate french battle_00_02_end_f96773ec:
 
     # sk nulla "Your arm...?!"
-    sk nulla ""
+    sk nulla "Votre bras... ?!"
 
 # game/script/story/st_00.rpy:532
 translate french battle_00_02_end_da8f53fc:
 
     # d nulla "Lost the old one during the Fall of Beacon.{w=0.3} This one's pretty useful, though, wouldn't you agree?"
-    d nulla ""
+    d nulla "J'ai perdu l'ancien durant la Chute de Beacon.{w=0.3} Mais celui là est plutôt utile, n'est-ce pas ?"
 
 # game/script/story/st_00.rpy:535
 translate french battle_00_02_end_d149f6db:
 
     # adt ".{w=0.3}.{w=0.3}.{w=0.5}Beacon...{w=0.5} Now I remember."
-    adt ""
+    adt ".{w=0.3}.{w=0.3}.{w=0.5}Beacon...{w=0.5} Je me souviens, maintenant."
 
 # game/script/story/st_00.rpy:539
 translate french battle_00_02_end_b56ffcc1:
 
     # adt "You're that kid...{w=0.5} {b}The one who got torn to shreds in the finals{/b}."
-    adt ""
+    adt "Tu es ce gamin...{w=0.5} {b}Celui qui s'est fait déchiqueter en finale{/b}."
 
 # game/script/story/st_00.rpy:546
 translate french battle_00_02_end_bb2267ea:
 
     # adt "But how...?"
-    adt ""
+    adt "Mais comment... ?"
 
 # game/script/story/st_00.rpy:547
 translate french battle_00_02_end_b56b5867:
 
     # d ".{w=0.3}.{w=0.3}.{w=0.5}as much as I'd love to make you pay for your involvement...{w=0.5} {i}Butcher of Beacon{/i}..."
-    d ""
+    d ".{w=0.3}.{w=0.3}.{w=0.5} Autant j'adorerai te faire payer pour ton implication...{w=0.5} {i}Boucher of Beacon{/i}..."
```

Comment:
Autant que j'adorerai te...

Reply:
Je fix. j'ai aussi remarqué le "of" -> "de"